scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -1344,8 +1344,12 @@ class Router(formatOps: FormatOps) {
       case FormatToken(_, c: T.Comment, between) =>
         Seq(Split(newlines2Modification(between), 0))
       // Commented out code should stay to the left
-      case FormatToken(c: T.Comment, _, between) if c.syntax.startsWith("//") =>
-        Seq(Split(Newline, 0))
+      case FormatToken(c: T.Comment, right, _) if c.syntax.startsWith("//") =>
```

Comment:
Could you add some comments that explain the behavior with some code examples? So that people who read this code can easily understand the meaning.

Reply:
will do